contrib/elasticsearch/readme.md

```diff
@@ -0,0 +1,56 @@
+
+# ELK Samples
+
+Helpful setup guide: https://www.elastic.co/guide/en/kibana/8.3/docker.html
+
+Note: These commands are meant to be run from the root of the Emissary project 
+
+## Elasticsearch and Kibana 
+
+Run Elasticsearch:
+```shell
+docker network create elastic
+docker run --name elasticsearch --hostname elasticsearch --net elastic -p 9200:9200 -p 9300:9300 -t docker.elastic.co/elasticsearch/elasticsearch:8.3.2
```

Comment:
It might be best to specify -e discovery.type=single-node for development use. 
```suggestion
docker run --name elasticsearch --hostname elasticsearch --net elastic -p 9200:9200 -p 9300:9300 -e discovery.type=single-node -t docker.elastic.co/elasticsearch/elasticsearch:8.3.2
```

Otherwise many systems will need to have the vm.max_map_count increased from the default system setting.